app/event/add/page.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+import {
+  Box,
+  Button,
+  Center,
+  Input,
+  Stack,
+  Select,
+  Text,
+} from "@chakra-ui/react";
+import { useState } from "react";
+export default function eventAdd() {
+  const [eventname, setEventname] = useState("");
+  const [date, setDate] = useState("");
+  const [range, setRange] = useState("option1");
+  return (
+    <Center>
+      <Stack  w={["80%", "50%"]} gap="5">
+        <Box fontSize="30px" textAlign="center">
+          イベントを作成する
+        </Box>
+        <Stack>
+          <Text fontSize="15px">イベント名</Text>
+          <Input
+            placeholder="HTML講習会"
+            value={eventname}
+            onChange={(e) => setEventname(e.target.value)}
+          />
+        </Stack>
+        <Stack>
+          <Text fontSize="15px">日時</Text>
+          <Input
+            type="date"
+            value={date}
+            onChange={(e) => setDate(e.target.value)}
+          />
+        </Stack>
+        <Stack>
+          <Text fontSize="15px">公開範囲</Text>
+          <Select
+            placeholder="一般公開"
+            value={range}
+            onChange={(e) => setRange(e.target.value)}
+          >
+            <option value="option2">非公開</option>
+            <option value="option3">限定公開</option>
+          </Select>
```

Comment:
プルダウンの使い方としては，以下のような実装のほうがわかりやすいかも！
公開範囲を必ず何かしら選択してもらえるように，選択しないとエラーを表示するとかね！
```suggestion
          <Select
            placeholder="公開範囲を選択してください"
            value={range}
            onChange={(e) => setRange(e.target.value)}
          >
            <option value="option1">非公開</option>
            <option value="option2">部内に公開</option>
            <option value="option3">一般に公開</option>
          </Select>
```

Reply:
setしたときのvalueがoption1だと扱いづらいとおもうのでそこも変更を忘れずに

Reply:
修正しました